specification/columns/chargeclassification.md

```diff
@@ -0,0 +1,42 @@
+# Charge Classification
+
+A Charge Classification indicates whether the row represents a regular charge or a correction to one or more previous charges.

```

Comment:
I don't view a "Credit" or "Tax" as a regular charge or 
a correction, so maybe ... "a regular, supporting, one-time, or correcting charge to one or more previous previous charges."

Reply:
this category is currently somewhat limited as we are trying to support two modes of refund, ideally we would like refunds to be included under the 'usage' category however we need to identify bulk adjustments also

Reply:
I'm interpreting Chris's comment as "Regular" not being a great term to describe the typical charges we see. I definitely feel that way. Some potential alternatives:
- Standard (may be confused with PricingCategory)
- Baseline
- Base
- Original
- Conventional

Reply:
Hey this adjustment was made and approved for merge on the previous joint WG call, these suggestions could be opened as an issue